cmd/gen-go-sample/main_test.go

```diff
@@ -48,9 +49,9 @@ func TestUnary(t *testing.T) {
 				`resource_field%bar="mybar"`,
 			},
 			Attributes: []SampleAttribute{
-				{"a.x", "the_x"},
-				{"b", "the_b"},
-				{"resource_field%foo", "the_foo"},
+				{"a.x", "the_x", false},
```

Comment:
Since the keyword `false` is not descriptive by itself, I think we should add field name here: `{Parameter: "a.x", SampleArgumentName: "the_x"}` (false is zero value for bool, so it can be skipped).